status/worksets.py

```diff
@@ -99,6 +99,28 @@ def get_workset_data(application, workset):
             for project in result[key]['projects']:
                 result[key]['projects'][project]['samples'] = dict(sorted(result[key]['projects'][project]['samples'].items()))
         return result
+class ClosedWorksetsHandler(SafeHandler):
+    """ Handles all closed worksets for the closed ws tab
+    URL: /api/v1/closed_worksets
+    """
+    def get(self):
+        result={}
+        a_year_ago = datetime.datetime.now() - relativedelta(years=1)
+        ws_view= self.application.worksets_db.view("worksets/summary", descending=True)
+        for row in ws_view:
+            try:
+                if parse(row.value['date_run']) <= a_year_ago:
+                    flag=True
+                    for project in row.value['projects']:
+                        if row.value['projects'][project]['status'] == 'Open':
+                            flag=False
+                            break
+                    if flag:
+                            result[row.key]=row.value
```

Comment:
Look great! I'd just like some more spacing around the operators here, like `result[row.key]=row.value` -> `result[row.key] = row.value`. Sorry for being picky 😄

Reply:
Good catch! :-)